pkgs/applications/file-managers/krusader/default.nix

```diff
@@ -15,13 +15,18 @@
 
 mkDerivation rec {
   pname = "krusader";
-  version = "2.7.2";
+  version = "2.8.0";
 
   src = fetchurl {
     url = "mirror://kde/stable/${pname}/${version}/${pname}-${version}.tar.xz";
-    hash = "sha256-QaOaQ7PELdHR7K6obfMMr/agYf7MHWb2CFmyo8qXYQk=";
+    hash = "sha256-jkzwWpMYsLwbCUGBG5iLLyuwwEoNHjeZghKpGQzywpo=";
   };
 
+  patches = [
+    # Fix compilation error due to forceful header include
```

Comment:
This feels a bit hacky because we do have `<KCompletion/kcompletion_version.h>` and `<KArchive/karchive_version.h>` in their respective derivations, but they don't seem to be picked up by the build.

Reply:
They do get picked up in the build, but only if KCompletion/karchive is specified as dependency in CMake. The problem is compat.h is included in every file, even those that don't depend on karchive. In other distros this works because all headers are in one place. But in nix case CMake doesn't pass the needed `-I` option to find the header so it results in errors.